### PR TITLE
chore(judicial-system): Iterates the appeal behaviour in all case types

### PR DESCRIPTION
**Pull request by gudjong**
# Iterates the appeal behaviour in all case types

https://app.asana.com/0/1199153462262248/1201312869487959

## What

☝️ 

## Why

Iteration after user input

## Screenshots / Gifs

image
image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/formatters/rulingPdf.ts [hunk 1]
```diff
@@ -647,37 +654,38 @@ function constructInvestigationRulingPdf(
...
+    prosecutorAppeal = existingCase.prosecutorAppealAnnouncement ?? ''
+  }
+
+  if (prosecutorAppeal) {
+    doc.text(' ').text(prosecutorAppeal, { align: 'justify', paragraphGap: 1 })
+  }
+
+  let accusedAppeal = formatAppeal(
+    existingCase.accusedAppealDecision,
+    'Varnaraðili',
```
Comment (telmag92) [inline apps/judicial-system/backend/src/app/formatters/rulingPdf.ts:676]: Wondering if 'Varnaraðili' as well as the strings in formatAppeal could be added to CF?

File: apps/judicial-system/backend/src/app/formatters/rulingPdf.ts [hunk 2]
```diff
@@ -647,37 +654,38 @@ function constructInvestigationRulingPdf(
...
 
-  // Only show accused appeal decision if applicable
-  if (
-    existingCase.accusedAppealDecision !== CaseAppealDecision.NOT_APPLICABLE
-  ) {
-    doc
-      .text(' ')
-      .text(
-        `${formatAppeal(existingCase.accusedAppealDecision, 'Varnaraðili')} ${
-          existingCase.accusedAppealDecision === CaseAppealDecision.APPEAL
```
Comment (telmag92) [inline apps/judicial-system/backend/src/app/formatters/rulingPdf.ts:672]: Import CaseAppealDecision is unused now. Might want to remove that :)

**Review by telmag92 — COMMENTED** [review]
Looks good, few minor comments

File: apps/judicial-system/backend/src/app/formatters/rulingPdf.ts (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/formatters/rulingPdf.ts:676]: They could, but I decided not to add them at this time. Added so many others on the client side. I would like to leave this for another day.

File: apps/judicial-system/backend/src/app/formatters/rulingPdf.ts (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/formatters/rulingPdf.ts:672]: Will fix.

**Review by telmag92 — APPROVED** [review]

**Pull request merged**